new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItInitContainers.java

```diff
@@ -0,0 +1,443 @@
+                    .runtimeEncryptionSecret(encryptionSecretName))));
+
+
+    switch (testCaseName) {
+      case "spec":
+        domain.getSpec().getServerPod().addInitContainersItem(new V1Container()
+            .addCommandItem("echo").addArgsItem("\"Hi from Domain\"")
+            .name("busybox")
+            .imagePullPolicy("IfNotPresent")
+            .image("busybox"));
+        break;
+      case "adminServer":
+        domain.getSpec().getAdminServer().serverPod(new ServerPod()
+            .addInitContainersItem(new V1Container()
+            .addCommandItem("echo").addArgsItem("\"Hi from AdminServer\"")
+            .name("busybox")
+            .imagePullPolicy("IfNotPresent")
+            .image("busybox")));
+        break;
+      case "clusters":
+        clusters = domain.getSpec().getClusters();
+        assertNotNull(clusters, "Can't find clusters in CRD ");
+        Cluster mycluster = clusters.stream()
+            .filter(cluster -> clusterName.equals(cluster.getClusterName())).findAny()
+            .orElse(null);
+        assertNotNull(mycluster, "Can't find cluster " + clusterName);
+        mycluster.serverPod(new ServerPod()
+                .addInitContainersItem(new V1Container()
+                    .addCommandItem("echo").addArgsItem("\"Hi from Cluster \"")
+                    .name("busybox")
+                    .imagePullPolicy("IfNotPresent")
+                    .image("busybox")));
+        break;
+      case "managedServers":
+        domain.getSpec().addManagedServersItem(new ManagedServer()
+            .serverName("managed-server1")
+            .serverPod(new ServerPod()
+                .addInitContainersItem(new V1Container()
+                    .addCommandItem("echo").addArgsItem("\"Hi from managed-server1\"")
+                    .name("busybox")
+                    .imagePullPolicy("IfNotPresent")
+                    .image("busybox"))));
+        break;
+      default:
+        logger.info("no match for provided case {0}", testCaseName);
+    }
+
+    // create model in image domain
+    logger.info("Creating model in image domain {0} in namespace {1} using docker image {2}",
+        domainUid, domainNamespace, miiImage);
+    createDomainAndVerify(domain, domainNamespace);
+    String adminServerPodName = domainUid + adminServerPrefix;
+    //check if pod in init state
+    checkPodInitializing(adminServerPodName,domainUid, domainNamespace);
+
+    // check that admin server pod exists in the domain namespace
+    logger.info("Checking that admin server pod {0} exists in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodExists(adminServerPodName, domainUid, domainNamespace);
+
+    // check that admin service exists in the domain namespace
+    logger.info("Checking that admin service {0} exists in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkServiceExists(adminServerPodName, domainNamespace);
```

Comment:
move this check before checkPodExists

Reply:
done

Reply:
Use the test utility CommonTestUtils.checkPodReadyAndServiceExists to combine the both operation.

Reply:
done